kafka/web/Dockerfile

```diff
@@ -1,3 +1,8 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
I think it's best to take the copyright out of here since we didn't write this file

Reply:
no longer applies